src/gentropy/dataset/l2g_features/other.py

```diff
@@ -81,6 +81,51 @@ def common_genecount_feature_logic(
     )
 
 
+def common_protein_coding_feature_logic(
```

Comment:
Since this is only going to be one feature, I don't see the need in having a `common` function.